lib/plugins/jexlParser.js

```diff
@@ -53,7 +53,7 @@ jexl.addTransform('urldecode', (val) => decodeURI(val));
 jexl.addTransform('replacestr', (str, from, to) => str.replace(from, to));
 jexl.addTransform('replaceregexp', (str, reg, to) => str.replace(new RegExp(reg), to));
 jexl.addTransform('replaceallstr', (str, from, to) => str.replaceAll(from, to));
-jexl.addTransform('replaceallregexp', (str, reg, to) => str.replaceAll(new RegExp(reg), to));
+jexl.addTransform('replaceallregexp', (str, reg, to) => str.replaceAll(new RegExp(reg,"g"), to));
```

Comment:
doc/expressionLanguage.md should be sync with this change

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1058/commits/56e12a9118579c553f7748c46fe332b11369d359